wid/gap.py

```diff
@@ -954,6 +954,18 @@ def hdl_wid_402(desc):
     return True
 
 
+def hdl_wid_402(desc):
+    #Please perform the Directed Connection Establishment Procedure using RPA
+    #then resolve the PTS address and connect with PTS.
+    stack = get_stack()
+
+    if not stack.gap.iut_has_privacy():
+        return False
+
+    btp.gap_conn()
```

Comment:
I think this does not test DCEP. Under the hood, this sends a Gap Connect command to IUT with PTS address as an argument.